.htmllintrc

```diff
@@ -0,0 +1,70 @@
+{
+  "attr-bans": [
+    "align",
+    "background",
+    "bgcolor",
+    "border",
+    "dynsrc",
+    "id",
+    "frameborder",
+    "longdesc",
+    "lowsrc",
+    "onclick",
+    "ondblclick",
+    "onload",
+    "marginwidth",
+    "marginheight",
+    "scrolling",
+    "style",
+    "width"
+  ],
+  "attr-name-ignore-regex": false,
+  "attr-name-style": "dash",
+  "attr-no-dup": true,
+  "attr-no-unsafe-chars": true,
+  "attr-quote-style": "double",
+  "attr-req-value": true,
+  "class-no-dup": true,
+  "class-style": "dash",
+  "csslint": false,
+  "doctype-first": false,
+  "doctype-html5": false,
+  "fig-req-figcaption": true,
+  "focusable-tabindex-style": false,
+  "head-req-title": true,
+  "href-style": false,
+  "html-req-lang": true,
+  "id-class-ignore-regex": false,
+  "id-class-no-ad": true,
+  "id-class-style": "underscore",
+  "id-no-dup": true,
+  "input-radio-req-name": true,
+  "img-req-alt": true,
+  "img-req-src": true,
+  "indent-style": "spaces",
+  "indent-width": 2,
+  "indent-width-cont": true,
+  "input-req-label": false,
+  "label-req-for": true,
+  "lang-style": "case",
+  "line-end-style": "lf",
+  "line-max-len": 120,
+  "line-max-len-ignore-regex": "/href/g",
+  "page-title": true,
+  "spec-char-escape": true,
+  "table-req-caption": false,
+  "table-req-header": true,
+  "tag-bans": [
+    "b",
+    "i",
+    "keygen",
+    "style"
+  ],
+  "tag-close": true,
+  "tag-name-lowercase": true,
+  "tag-name-match": true,
+  "tag-self-close": "never",
+  "text-escape-spec-char": true,
+  "title-max-len": 60,
```

Comment:
@iondrimba same of our pages have more than 60 letters (sample: https://www.getninjas.com.br/reformas-e-reparos/pedreiro/sp/sao-paulo/orcamentos/4485663-preciso-reformar-uma-casa-antiga-e).

120 could be a nice number